eProctoring/user/views.py

```
@@ -1,4 +1,71 @@
+            messages.success(request, "Registration successful. You can now log in.")
+            return redirect('login')  # Redirect to login page after successful registration
+        else:
+            messages.error(request, "Registration failed. Please correct the errors below.")
+    else:
+        form = RegisterForm()
+    
+    return render(request, 'registration/register.html', {'form': form})
+
+
+
+def logout_view(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `logout_view`

(ANN201)